setup.py

```diff
@@ -32,5 +25,5 @@ def read(fname):
         "Programming Language :: Python :: 2",
         "Programming Language :: Python :: 3",
     ],
-    requires=['h5py', 'numpy', 'mongoengine', 'jsonschema']
+    # requires=['h5py', 'numpy', 'mongoengine', 'jsonschema'],
```

Comment:
let's try taht again...

IIRC `requires` is not a valid kwarg for `setup()`. Also I'm pretty sure this is the comprehensive list: https://github.com/jaraco/setuptools/blob/master/setup.py#L95